lib/pages/home_page.dart

```diff
@@ -34,7 +34,7 @@ class _HomePageState extends State<HomePage> {
                 var now = new DateTime.now();
                 List<Reminder> listFromBox = reminders.values
                     .where((reminder) =>
-                        (now.difference(reminder.dateTime!).inDays == 0))
+                        (now.difference(reminder.dateTime ?? now).inDays == 0))
```

Comment:
This looks like the reminders will be hidden after one full day.
if the date of reminder is 2 Oct 12:10. and ```now``` is 2 Oct 12:15. the difference will be ```zero```